src/test/groovy/com/github/benmanes/gradle/versions/BaseSpecification.groovy

```diff
@@ -9,15 +9,7 @@ class BaseSpecification extends Specification {
   File buildFile
   List<File> pluginClasspath
 
-  def setup() {
+  def 'setup'() {
     buildFile = testProjectDir.newFile('build.gradle')
-
```

Comment:
We only need some of this code for a single test, the `DifferentGradleVersionsSpec`.